IrConsoleApp/Lecture6/Course.cs

```diff
@@ -0,0 +1,78 @@
+﻿namespace IrynaZharnasek.Lecture6
+{
+    public class Course
+    {
+        private string course;
+        private Mentor mentor;
```

Comment:
mentor is unused, you need to sync logic: if you expected that 1 course should have 1 mentor, that you need to keep this variable and remove variable arrayOfMentors. If you expected that 1 course should have several mentors, that you need to remove mentor variable and keep arrayOfMentors, up to you

Reply:
Done